### DISCUSSION TITLE
Support Piano Chords and Scales

### DISCUSSION THREAD

**Discussion by sebastian-beca** [olvidalo/obsidian-chord-sheets#18] [Ideas]
Wondering if possible to add support for Piano as another instrument and display chord diagrams and inversions / alternate voicings 🎹 
Perhaps this involves modifying /forking vexchords or using something like this:
https://github.com/amypellegrini/piano-chord-charts

I can code a little bit wondering if you’ve already worked on this or have it planned?

**Comment by emilionavas** [comment]
Adding piano support for this plugin could be something wonderful. 
Are you planning to add piano support chors diagrams?

**Comment by olvidalo** [comment]
Piano diagrams are now available in the newest release
